src/Analyser/NodeScopeResolver.php

```diff
@@ -466,6 +472,12 @@ private function processStmtNode(
 					new StatementExitPoint($stmt, $scope),
 				]);
 			}
+		} elseif ($stmt instanceof FuncCall && count($this->earlyTerminatingFunctionCalls) > 0) {
```

Comment:
FuncCall is not a statement, but an expression. See how MethodCall is handled here in this class and do the same for FuncCall.